dockerfiles/framework/scale/deployDb.py

```diff
@@ -22,6 +21,10 @@ def run():
     else:
         cfg['volumes'] = '{"containerPath": "/var/lib/pgsql/data","hostPath": "%(dbHostVol)s","mode": "RW"}' % cfg
 
+    if os.environ.get('INIT_DB', False):
+        # attempt to delete an old instance..if it doesn't exists it will error but we don't care so we ignore it
+        requests.delete('http://marathon.mesos:8080/v2/apps/' + cfg['scaleDBHost'])
+
```

Comment:
I didn't see value in doing this on INIT_DB or DEPLOY_DB being `True`. Feel free to argue otherwise, but I don't really anticipate a normal user setting DEPLOY_DB to True and remembering to set it to False between restarts or scaling in Marathon. If so, your database would get wiped. 😦  As a result I removed this entire block. The DB connections are already being maintained between scheduler restarts, so the issue that affected log stash doesn't apply here.